### ISSUE TITLE
(@aws-cdk-lib/aws_cloudfront.CloudFrontWebDistribution) Link to Typescript source is broken

### ISSUE THREAD

**Issue by its-mirus-lu** [aws/aws-cdk#19733]
### Describe the issue

TL;DR: The Typescript `source` link for CloudFrontWebDistribution API documentation is broken.

 At the top of the API documentation page for CloudFrontWebDistribution is a table with links to each of the supported languages. Next to Typescript is the word `source` in brackets which is a link to the source code for the given class.  In this case the link points to a location that no longer exists (most likely due to the v1 to v2 CDK migration).

### Links

Here is the API doc page with the broken `source` link https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudfront.CloudFrontWebDistribution.html

Here is where current URL that the link points to: [https://github.com/aws/aws-cdk/blob/v2.19.0/packages/aws-cdk-lib/aws-cloudfront/lib/web-distribution.ts#L744](https://github.com/aws/aws-cdk/blob/v2.19.0/packages/aws-cdk-lib/aws-cloudfront/lib/web-distribution.ts#L744)

Here is where the link should point to: [https://github.com/aws/aws-cdk/blob/3c9ea5f31e3113fb0d2ba5c633fcd665294a70eb/packages/%40aws-cdk/aws-cloudfront/lib/web-distribution.ts ](https://github.com/aws/aws-cdk/blob/3c9ea5f31e3113fb0d2ba5c633fcd665294a70eb/packages/%40aws-cdk/aws-cloudfront/lib/web-distribution.ts ) (or with a path change to the latest tagged release version)

**Comment by peterwoodworth** [comment]
hey @its-mirus-lu,

Thanks for reporting this 🙂 We're aware of the problem - the links to the code on v2 docs don't work because the v2 code doesn't actually lie anywhere - it's autogenerated based on the v1 code.

We don't have an ETA on a fix, but for now you can go to the identical v1 docs page and get the link to the source code that way
